### ISSUE TITLE
Carboard application using WebVR

### ISSUE THREAD

**Issue by ondrabus** [Kentico/community-content-backlog#13]
### Motivation

The emergence of the WebVR API is one of the fireworks moment that opens a whole new world of exciting possibilities. The WebVR API empowers web developers to build virtual-reality experiences with web technologies and also enable users to have that fully immersed experience using just their browser.

https://developer.mozilla.org/en-US/docs/Web/API/WebVR_API
https://auth0.com/blog/getting-started-with-the-new-webvr-api/

### Proposed idea

Create an app using WebVR and content from KC, write an article about it.

**Comment by kentico-ericd** [comment]
I have a bit of a working prototype going, but quick question- does it have to be Google Cardboard? The problem is, Cardboard doesn't have any inputs, so the user is limited to just looking around, which limits what kind of integration can be made. Can the integration be for another VR headset?

**Comment by petrsvihlik** [comment]
Absolutely. Initially, we thought it'd be easier to have something for cardboard because it's affordable and anyone can test it...but feel free to use another VR headset.
